### PR TITLE
update document.mk and ug.tex

### PR DESCRIPTION
**Pull request by codingUniv**

### CODE REVIEW

File: document/document.mk [hunk 1]
```diff
@@ -3,6 +3,7 @@ SHELL = /bin/bash
 LIB_DOC_DIR:=$(LIB_DIR)/document
 LIB_SW_DIR:=$(LIB_DIR)/software
 LIB_SW_PYTHON_DIR:=$(LIB_SW_DIR)/python
+PARAM_SYNTH ?=1
```
Comment (jjts) [inline document/document.mk:6]: if you do not put any //PARAM OR //MACRO pragmas it should not generate the section. I see no use for this variable.

Comment (codingUniv) [reply document/document.mk:6]: For some IP cores, there was an interest to insert the parameters and macros tables in a section different from the Synthesis section. To be able to do that while keeping other repositories intact, this solution has been suggested.

If interested in (e.g. iob-cache), it is enough to set PARAM_SYNTH to a different value in <IP_ROOT_DIR>/document/document.mk and customize custom.tex as needed. For others, there is nothing to change.

Comment (jjts) [reply document/document.mk:6]: We used these variables before, and we gave them up as code comments, and not makefile variables should entirely drive the process. This objective may is not wholly achieved but should be the direction to follow. If we add this sort of variable, we will be going back.

If I remember correctly, the only problem with inserting these parameters in the synthesis section is that they are referred to before they are described. I suggest keeping them in the synthesis section and solving this problem later.

**Comment by codingUniv** [comment]
Using lscape package is always suggested. Please refer to the new commit 8e7707c

File: document/document.mk (hunk 1 above)
Comment (codingUniv) [reply document/document.mk]: Modification canceled taken into account your feedback and last structure's updates.

**Pull request merged**